### ISSUE TITLE
Python error with docker-compose build

### ISSUE THREAD

**Issue by AhmadShiddiq** [laradock/laradock#2429]
### Info:
- Docker version (`$ docker --version : Docker version 19.03.5-ce, build 633a0ea838`): 
- Laradock commit (`$ git rev-parse HEAD : e943d90ca28b61b7798abd2f276fdaa44a39106f`): 
- System info (Linux): 
- System info disto/version: Manjaro Linux

### Issue:
After running `docker-compose up -d mysql phpmyadmin  nginx redis` i got info to build first.
_____

### Expected behavior:
Something error with python.
_____

### Reproduce:
After fresh install run this `docker-compose build`
_____

### Relevant Code:

```
Step 5/18 : RUN curl -O https://bootstrap.pypa.io/get-pip.py &&     python3 get-pip.py &&     rm get-pip.py
 ---> Running in 5642efa9bbfa
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed                                                                                           
100 1734k  100 1734k    0     0  63982      0  0:00:27  0:00:27 --:--:-- 74055                                                                                          
Traceback (most recent call last):                                                                                                                                      
  File "get-pip.py", line 22318, in <module>                                                                                                                            
    main()                                                                                                                                                              
  File "get-pip.py", line 198, in main                                                                                                                                  
    bootstrap(tmpdir=tmpdir)                                                                                                                                            
  File "get-pip.py", line 82, in bootstrap                                                                                                                              
    import pip._internal.main                                                                                                                                           
  File "/tmp/tmp2wiih3bp/pip.zip/pip/_internal/main.py", line 13, in <module>                                                                                           
  File "/tmp/tmp2wiih3bp/pip.zip/pip/_internal/cli/autocompletion.py", line 11, in <module>                                                                             
  File "/tmp/tmp2wiih3bp/pip.zip/pip/_internal/cli/main_parser.py", line 7, in <module>                                                                                 
  File "/tmp/tmp2wiih3bp/pip.zip/pip/_internal/cli/cmdoptions.py", line 19, in <module>                                                                                 
ModuleNotFoundError: No module named 'distutils.util'                                                                                                                   
ERROR: Service 'jupyterhub-user' failed to build: The command '/bin/bash -c curl -O https://bootstrap.pypa.io/get-pip.py &&     python3 get-pip.py &&     rm get-pip.py' 
returned a non-zero code: 1
```

**Comment by sergebenard** [comment]
Seemingly the same issue

- Docker version ($ docker --version : `Docker version 19.03.5-ce, build 633a0ea`):
- Laradock commit ($ git rev-parse HEAD : `d732318e5286f6e06beb5d1c958a9ecaee4fe1b5`):
- System info (`Windows 10 Pro`):
- System info disto/version: `64 bit, Version 10.0.18362, Build 18362`

### Relevant Code:

`c:\laradock>docker-compose build
pgadmin uses an image, skipping
gitlab-runner uses an image, skipping
docker-in-docker uses an image, skipping
blackfire uses an image, skipping
netdata uses an image, skipping
metabase uses an image, skipping
mailu-front uses an image, skipping
mailu-imap uses an image, skipping
mailu-smtp uses an image, skipping
mailu-antispam uses an image, skipping
mailu-antivirus uses an image, skipping
mailu-webdav uses an image, skipping
mailu-admin uses an image, skipping
mailu-webmail uses an image, skipping
mailu-fetchmail uses an image, skipping
mailu uses an image, skipping
confluence uses an image, skipping
tomcat uses an image, skipping
Building minio
Step 1/3 : FROM minio/minio
 ---> caf8e214f2c4
Step 2/3 : LABEL maintainer="Thor Erik Lie <thor@thorerik.com>"
 ---> Using cache
 ---> 1dc943413b3b
Step 3/3 : ENTRYPOINT ["minio", "server", "/export"]
 ---> Using cache
 ---> 67a3349c5a92
Successfully built 67a3349c5a92
Successfully tagged laradock_minio:latest
Building mysql
Step 1/10 : ARG MYSQL_VERSION=latest
Step 2/10 : FROM mysql:${MYSQL_VERSION}
 ---> d435eee2caa5
Step 3/10 : LABEL maintainer="Mahmoud Zalt <mahmoud@zalt.me>"
 ---> Using cache
 ---> 78bcb0c34abc
Step 4/10 : ARG TZ=UTC
 ---> Using cache
 ---> acef54257681
Step 5/10 : ENV TZ ${TZ}
 ---> Using cache
 ---> 9d22a43aac7a
Step 6/10 : RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone && chown -R mysql:root /var/lib/mysql/
 ---> Using cache
 ---> c4ff4b3490d6
Step 7/10 : COPY my.cnf /etc/mysql/conf.d/my.cnf
 ---> Using cache
 ---> 12f67fbff00d
Step 8/10 : RUN chmod 0444 /etc/mysql/conf.d/my.cnf
 ---> Using cache
 ---> fb4339185af0
Step 9/10 : CMD ["mysqld"]
 ---> Using cache
 ---> ba472901fdd9
Step 10/10 : EXPOSE 3306
 ---> Using cache
 ---> ae0e09ae25b3
Successfully built ae0e09ae25b3
Successfully tagged laradock_mysql:latest
Building percona
Step 1/6 : FROM percona:5.7
 ---> 2e7138331e55
Step 2/6 : LABEL maintainer="DTUNES <diegotdai@gmai.com>"
 ---> Using cache
 ---> e026f9506bed
Step 3/6 : RUN chown -R mysql:root /var/lib/mysql/
 ---> Using cache
 ---> af355bbe5547
Step 4/6 : COPY my.cnf /etc/mysql/conf.d/my.cnf
 ---> Using cache
 ---> 93c1e845967a
Step 5/6 : CMD ["mysqld"]
 ---> Using cache
 ---> 4dbf15ec374d
Step 6/6 : EXPOSE 3306
 ---> Using cache
 ---> 6fe1cf4c4f9c
Successfully built 6fe1cf4c4f9c
Successfully tagged laradock_percona:latest
Building mssql
Step 1/12 : FROM microsoft/mssql-server-linux
 ---> 314918ddaedf
Step 2/12 : LABEL maintainer="Mahmoud Zalt <mahmoud@zalt.me>"
 ---> Using cache
 ---> 5288da2a4d7c
Step 3/12 : RUN mkdir -p /usr/src/app
 ---> Using cache
 ---> 8ff6721ba1c2
Step 4/12 : WORKDIR /usr/src/app
 ---> Using cache
 ---> dc47a4a9d107
Step 5/12 : COPY . /usr/src/app
 ---> Using cache
 ---> 70873e82dc3e
Step 6/12 : RUN chmod +x /usr/src/app/create_table.sh
 ---> Using cache
 ---> db6ed386f059
Step 7/12 : ENV MSSQL_DATABASE=$MSSQL_DATABASE
 ---> Using cache
 ---> bf17c2a1f8d5
Step 8/12 : ENV ACCEPT_EULA=Y
 ---> Using cache
 ---> 9d317a362930
Step 9/12 : ENV SA_PASSWORD=$MSSQL_PASSWORD
 ---> Using cache
 ---> 6142abd5a481
Step 10/12 : VOLUME /var/opt/mssql
 ---> Using cache
 ---> 7112c4e63583
Step 11/12 : EXPOSE 1433
 ---> Using cache
 ---> 6d622278cf2f
Step 12/12 : CMD /bin/bash ./entrypoint.sh
 ---> Using cache
 ---> 0b3622eb5097
Successfully built 0b3622eb5097
Successfully tagged laradock_mssql:latest
Building mariadb
Step 1/10 : ARG MARIADB_VERSION=latest
Step 2/10 : FROM mariadb:${MARIADB_VERSION}
 ---> e93d99aa9076
Step 3/10 : LABEL maintainer="Mahmoud Zalt <mahmoud@zalt.me>"
 ---> Using cache
 ---> 61562e37a63c
Step 4/10 : ARG TZ=UTC
 ---> Using cache
 ---> f46a14638cc7
Step 5/10 : ENV TZ ${TZ}
 ---> Using cache
 ---> f932f91ad3d7
Step 6/10 : RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone && chown -R mysql:root /var/lib/mysql/
 ---> Using cache
 ---> c5d6e1ec2db2
Step 7/10 : COPY my.cnf /etc/mysql/conf.d/my.cnf
 ---> Using cache
 ---> 9758d1176131
Step 8/10 : RUN chmod -R 644 /etc/mysql/conf.d/my.cnf
 ---> Using cache
 ---> df0235d5e456
Step 9/10 : CMD ["mysqld"]
 ---> Using cache
 ---> a1568ca8a513
Step 10/10 : EXPOSE 3306
 ---> Using cache
 ---> b873c83dc580
Successfully built b873c83dc580
Successfully tagged laradock_mariadb:latest
Building postgres
Step 1/4 : FROM postgres:alpine
 ---> c8abd8e13a5c
Step 2/4 : LABEL maintainer="Ben M <git@bmagg.com>"
 ---> Using cache
 ---> bef6ff0aec05
Step 3/4 : CMD ["postgres"]
 ---> Using cache
 ---> ceaa0a1046c2
Step 4/4 : EXPOSE 5432
 ---> Using cache
 ---> 4fa2e49d268f
Successfully built 4fa2e49d268f
Successfully tagged laradock_postgres:latest
Building postgres-postgis
Step 1/4 : FROM mdillon/postgis:latest
 ---> b2a8fefe8a63
Step 2/4 : LABEL maintainer="Mahmoud Zalt <mahmoud@zalt.me>"
 ---> Using cache
 ---> df98773db5bf
Step 3/4 : CMD ["postgres"]
 ---> Using cache
 ---> 6c9db44fba9f
Step 4/4 : EXPOSE 5432
 ---> Using cache
 ---> 52fa3bb4e54d
Successfully built 52fa3bb4e54d
Successfully tagged laradock_postgres-postgis:latest
Building neo4j
Step 1/4 : FROM tpires/neo4j
 ---> d2104ba4b832
Step 2/4 : LABEL maintainer="Mahmoud Zalt <mahmoud@zalt.me>"
 ---> Using cache
 ---> b382d4c18c2f
Step 3/4 : VOLUME /var/lib/neo4j/data
 ---> Using cache
 ---> c789cbb9542c
Step 4/4 : EXPOSE 7474 1337
 ---> Using cache
 ---> 412fcf92dd86
Successfully built 412fcf92dd86
Successfully tagged laradock_neo4j:latest
Building mongo
Step 1/5 : FROM mongo:latest
 ---> 9979235fc504
Step 2/5 : LABEL maintainer="Mahmoud Zalt <mahmoud@zalt.me>"
 ---> Using cache
 ---> 488d68486369
Step 3/5 : VOLUME /data/db /data/configdb
 ---> Using cache
 ---> fe2499c2a36f
Step 4/5 : CMD ["mongod"]
 ---> Using cache
 ---> 5a6f369f8264
Step 5/5 : EXPOSE 27017
 ---> Using cache
 ---> 35cdc3cbf991
Successfully built 35cdc3cbf991
Successfully tagged laradock_mongo:latest
Building rethinkdb
Step 1/7 : FROM rethinkdb:latest
 ---> 651856fa5c9d
Step 2/7 : LABEL maintainer="Cristian Mello <cristianc.mello@gmail.com>"
 ---> Using cache
 ---> 8b6cccee5a8f
Step 3/7 : VOLUME /data/rethinkdb_data
 ---> Using cache
 ---> ac886e83a51b
Step 4/7 : RUN apt-get -y update     && apt-get -y upgrade     && apt-get -y install python-pip     && pip install rethinkdb     && rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 902911befac3
Step 5/7 : RUN cp /etc/rethinkdb/default.conf.sample /etc/rethinkdb/instances.d/instance1.conf
 ---> Using cache
 ---> 22955d30915e
Step 6/7 : CMD ["rethinkdb", "--bind", "all"]
 ---> Using cache
 ---> 96155b90bf5c
Step 7/7 : EXPOSE 8080
 ---> Using cache
 ---> f6283ea9ae11
Successfully built f6283ea9ae11
Successfully tagged laradock_rethinkdb:latest
Building redis
Step 1/5 : FROM redis:latest
 ---> dcf9ec9265e0
Step 2/5 : LABEL maintainer="Mahmoud Zalt <mahmoud@zalt.me>"
 ---> Using cache
 ---> e5c8459f6891
Step 3/5 : VOLUME /data
 ---> Using cache
 ---> 60ad6e6a9b55
Step 4/5 : EXPOSE 6379
 ---> Using cache
 ---> fee229daf410
Step 5/5 : CMD ["redis-server"]
 ---> Using cache
 ---> cdac56e27d1d
Successfully built cdac56e27d1d
Successfully tagged laradock_redis:latest
Building redis-cluster
Step 1/2 : FROM grokzen/redis-cluster:latest
 ---> da1e58b65af0
Step 2/2 : LABEL maintainer="hareku <hareku908@gmail.com>"
 ---> Using cache
 ---> 3a5d7b177566
Successfully built 3a5d7b177566
Successfully tagged laradock_redis-cluster:latest
Building zookeeper
Step 1/6 : FROM zookeeper:latest
 ---> 611ffeaf5959
Step 2/6 : LABEL maintainer="Hyduan <hyduan96@qq.com>"
 ---> Using cache
 ---> 444c5d1a7c98
Step 3/6 : VOLUME /data
 ---> Using cache
 ---> d8f163fd73c8
Step 4/6 : VOLUME /datalog
 ---> Using cache
 ---> a05dd8a413a6
Step 5/6 : EXPOSE 2181
 ---> Using cache
 ---> 5695e9b1bb58
Step 6/6 : CMD ["zkServer.sh", "start-foreground"]
 ---> Using cache
 ---> b27b531ea99d
Successfully built b27b531ea99d
Successfully tagged laradock_zookeeper:latest
Building aerospike
Step 1/2 : FROM aerospike:latest
 ---> b6d73b5d3b4d
Step 2/2 : LABEL maintainer="Luciano Jr <luciano@lucianojr.com.br>"
 ---> Using cache
 ---> b6587b8d071e
Successfully built b6587b8d071e
Successfully tagged laradock_aerospike:latest
Building phpmyadmin
Step 1/5 : FROM phpmyadmin/phpmyadmin
 ---> 5c0865e6f546
Step 2/5 : LABEL maintainer="Bo-Yi Wu <appleboy.tw@gmail.com>"
 ---> Using cache
 ---> aa4ce177fc41
Step 3/5 : VOLUME /sessions
 ---> Using cache
 ---> 839e31d2c0de
Step 4/5 : RUN echo '' >> /usr/local/etc/php/conf.d/php-phpmyadmin.ini  && echo '[PHP]' >> /usr/local/etc/php/conf.d/php-phpmyadmin.ini  && echo 'post_max_size = 2G' >> /usr/local/etc/php/conf.d/php-phpmyadmin.ini  && echo 'upload_max_filesize = 2G' >> /usr/local/etc/php/conf.d/php-phpmyadmin.ini
 ---> Using cache
 ---> 7627d9e3d979
Step 5/5 : EXPOSE 80
 ---> Using cache
 ---> 611fa0896575
Successfully built 611fa0896575
Successfully tagged laradock_phpmyadmin:latest
Building certbot
Step 1/6 : FROM phusion/baseimage:latest
 ---> 166cfc3f6974
Step 2/6 : LABEL maintainer="Mahmoud Zalt <mahmoud@zalt.me>"
 ---> Using cache
 ---> ddc330471408
Step 3/6 : COPY run-certbot.sh /root/certbot/run-certbot.sh
 ---> Using cache
 ---> 5261a44cde07
Step 4/6 : RUN apt-get update
 ---> Using cache
 ---> f29552d528c7
Step 5/6 : RUN apt-get install -y letsencrypt
 ---> Using cache
 ---> 0dc3ca5e28b4
Step 6/6 : ENTRYPOINT bash -c "bash /root/certbot/run-certbot.sh && sleep infinity"
 ---> Using cache
 ---> 9321a716b697
Successfully built 9321a716b697
Successfully tagged laradock_certbot:latest
Building mailhog
Step 1/4 : FROM mailhog/mailhog
 ---> e00a21e210f9
Step 2/4 : LABEL maintainer="Mahmoud Zalt <mahmoud@zalt.me>"
 ---> Using cache
 ---> 9e416d43e396
Step 3/4 : CMD ["Mailhog"]
 ---> Using cache
 ---> 78f662a51277
Step 4/4 : EXPOSE 1025 8025
 ---> Using cache
 ---> 8f606f8e0a64
Successfully built 8f606f8e0a64
Successfully tagged laradock_mailhog:latest
Building maildev
Step 1/3 : FROM djfarrelly/maildev
 ---> 8c016b0ceb3c
Step 2/3 : LABEL maintainer="Maxime Hélias <maximehelias16@gmail.com>"
 ---> Using cache
 ---> 4fb667e1ff04
Step 3/3 : EXPOSE 80 25
 ---> Using cache
 ---> cfb857f95245
Successfully built cfb857f95245
Successfully tagged laradock_maildev:latest
Building selenium
Step 1/3 : FROM selenium/standalone-chrome
 ---> c022ef61b9a5
Step 2/3 : LABEL maintainer="Edmund Luong <edmundvmluong@gmail.com>"
 ---> Using cache
 ---> a7fd5ff4d5cc
Step 3/3 : EXPOSE 4444
 ---> Using cache
 ---> c8dce5f7a6b1
Successfully built c8dce5f7a6b1
Successfully tagged laradock_selenium:latest
Building jenkins
Step 1/39 : FROM openjdk:8-jdk
 ---> 09df0563bdfc
Step 2/39 : RUN apt-get update && apt-get install -y git curl && rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> e063577097ac
Step 3/39 : ENV JENKINS_HOME /var/jenkins_home
 ---> Using cache
 ---> 45d0bfbcc1b7
Step 4/39 : ENV JENKINS_SLAVE_AGENT_PORT 50000
 ---> Using cache
 ---> 0e85f6aab236
Step 5/39 : ARG user=jenkins
 ---> Using cache
 ---> a81a7e7af9bc
Step 6/39 : ARG group=jenkins
 ---> Using cache
 ---> b8a34efb4632
Step 7/39 : ARG uid=1000
 ---> Using cache
 ---> f373a25c5b2a
Step 8/39 : ARG gid=1000
 ---> Using cache
 ---> 233f36478d1d
Step 9/39 : RUN groupadd -g ${gid} ${group}     && useradd -d "$JENKINS_HOME" -u ${uid} -g ${gid} -m -s /bin/bash ${user}
 ---> Using cache
 ---> d0a5f494d700
Step 10/39 : VOLUME /var/jenkins_home
 ---> Using cache
 ---> 19cd13fab926
Step 11/39 : RUN mkdir -p /usr/share/jenkins/ref/init.groovy.d
 ---> Using cache
 ---> c2f2b5afafc7
Step 12/39 : ENV TINI_VERSION 0.16.1
 ---> Using cache
 ---> 8b3e42be7d34
Step 13/39 : ENV TINI_SHA d1cb5d71adc01d47e302ea439d70c79bd0864288
 ---> Using cache
 ---> b20b4132e5f1
Step 14/39 : RUN curl -fsSL https://github.com/krallin/tini/releases/download/v${TINI_VERSION}/tini-static-amd64 -o /bin/tini && chmod +x /bin/tini   && echo "$TINI_SHA  /bin/tini" | sha1sum -c -
 ---> Using cache
 ---> 68624a31afd4
Step 15/39 : COPY init.groovy /usr/share/jenkins/ref/init.groovy.d/tcp-slave-agent-port.groovy
 ---> Using cache
 ---> c8366d74fc80
Step 16/39 : ARG JENKINS_VERSION
 ---> Using cache
 ---> 471408d7eedd
Step 17/39 : ENV JENKINS_VERSION ${JENKINS_VERSION:-2.89.2}
 ---> Using cache
 ---> 79a3ce0d6fe1
Step 18/39 : ARG JENKINS_SHA=014f669f32bc6e925e926e260503670b32662f006799b133a031a70a794c8a14
 ---> Using cache
 ---> 3dd58adbe1a5
Step 19/39 : ARG JENKINS_URL=https://repo.jenkins-ci.org/public/org/jenkins-ci/main/jenkins-war/${JENKINS_VERSION}/jenkins-war-${JENKINS_VERSION}.war
 ---> Using cache
 ---> e5fac8cce062
Step 20/39 : RUN curl -fsSL ${JENKINS_URL} -o /usr/share/jenkins/jenkins.war   && echo "${JENKINS_SHA}  /usr/share/jenkins/jenkins.war" | sha256sum -c -
 ---> Using cache
 ---> 14f3666aa781
Step 21/39 : ENV JENKINS_UC https://updates.jenkins.io
 ---> Using cache
 ---> 5b44ffc8fdba
Step 22/39 : RUN chown -R ${user} "$JENKINS_HOME" /usr/share/jenkins/ref
 ---> Using cache
 ---> f30ce864d28e
Step 23/39 : ARG DOCKER_GID=998
 ---> Using cache
 ---> 3f119fa076fa
Step 24/39 : RUN groupadd -g ${DOCKER_GID} docker   && curl -sSL https://get.docker.com/ | sh   && apt-get -q autoremove   && apt-get -q clean -y   && rm -rf /var/lib/apt/lists/* /var/cache/apt/*.bin
 ---> Using cache
 ---> 28777ed86537
Step 25/39 : RUN apt-get install -y curl && curl -sSL https://get.docker.com/ | sh
 ---> Using cache
 ---> 6e4ab7ade10f
Step 26/39 : RUN usermod -aG docker jenkins
 ---> Using cache
 ---> a0e40853255e
Step 27/39 : RUN curl -L "https://github.com/docker/compose/releases/download/1.16.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
 ---> Using cache
 ---> 133fc7c9b780
Step 28/39 : RUN chmod +x /usr/local/bin/docker-compose
 ---> Using cache
 ---> 42005732ed02
Step 29/39 : EXPOSE 8080
 ---> Using cache
 ---> eb55482270cc
Step 30/39 : EXPOSE 50000
 ---> Using cache
 ---> 9c755ad9ea49
Step 31/39 : ENV COPY_REFERENCE_FILE_LOG $JENKINS_HOME/copy_reference_file.log
 ---> Using cache
 ---> b8954b1f6918
Step 32/39 : USER ${user}
 ---> Using cache
 ---> 45265eddd336
Step 33/39 : COPY jenkins-support /usr/local/bin/jenkins-support
 ---> Using cache
 ---> b16555d5e1cf
Step 34/39 : COPY jenkins.sh /usr/local/bin/jenkins.sh
 ---> Using cache
 ---> 2830702eabec
Step 35/39 : ENTRYPOINT ["/bin/tini", "--", "/usr/local/bin/jenkins.sh"]
 ---> Using cache
 ---> a6f21758cc4a
Step 36/39 : COPY plugins.sh /usr/local/bin/plugins.sh
 ---> Using cache
 ---> de0a13e2062e
Step 37/39 : COPY install-plugins.sh /usr/local/bin/install-plugins.sh
 ---> Using cache
 ---> c23a7d4c0a97
Step 38/39 : USER root
 ---> Using cache
 ---> a7c7b07e362d
Step 39/39 : RUN usermod -aG docker jenkins
 ---> Using cache
 ---> 6e81aa04c42b
Successfully built 6e81aa04c42b
Successfully tagged laradock_jenkins:latest
Building grafana
Step 1/2 : FROM grafana/grafana:latest
 ---> 7a40c3c56100
Step 2/2 : EXPOSE 3000
 ---> Using cache
 ---> 9e9549b4aa73
Successfully built 9e9549b4aa73
Successfully tagged laradock_grafana:latest
Building laravel-echo-server
Step 1/9 : FROM node:alpine
 ---> b850b4746cd9
Step 2/9 : RUN mkdir -p /usr/src/app
 ---> Using cache
 ---> 491497b5a402
Step 3/9 : WORKDIR /usr/src/app
 ---> Using cache
 ---> 0874fa97008b
Step 4/9 : COPY package.json /usr/src/app/
 ---> Using cache
 ---> ebbe4b7103a2
Step 5/9 : RUN apk add --update     python     python-dev     py-pip     build-base
 ---> Using cache
 ---> b938451a7ce5
Step 6/9 : RUN npm install
 ---> Using cache
 ---> 623e145cb0c2
Step 7/9 : COPY laravel-echo-server.json /usr/src/app/laravel-echo-server.json
 ---> Using cache
 ---> d3e52c5bcdde
Step 8/9 : EXPOSE 3000
 ---> Using cache
 ---> a4eeb1ef3c8e
Step 9/9 : CMD [ "npm", "start", "--force" ]
 ---> Using cache
 ---> a68d94741bcb
Successfully built a68d94741bcb
Successfully tagged laradock_laravel-echo-server:latest
Building solr
Step 1/8 : ARG SOLR_VERSION=5.5
Step 2/8 : FROM solr:${SOLR_VERSION}
 ---> 68a613a35fb2
Step 3/8 : ARG SOLR_DATAIMPORTHANDLER_MYSQL=false
 ---> Using cache
 ---> 92b054f67693
Step 4/8 : ENV SOLR_DATAIMPORTHANDLER_MYSQL ${SOLR_DATAIMPORTHANDLER_MYSQL}
 ---> Using cache
 ---> 5e9a12fcb374
Step 5/8 : RUN if [ ${SOLR_DATAIMPORTHANDLER_MYSQL} = true ]; then     curl -L -o /tmp/mysql_connector.tar.gz "https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-5.1.45.tar.gz"        && mkdir /opt/solr/contrib/dataimporthandler/lib        && tar -zxvf /tmp/mysql_connector.tar.gz -C /opt/solr/contrib/dataimporthandler/lib "mysql-connector-java-5.1.45/mysql-connector-java-5.1.45-bin.jar" --strip-components 1        && rm /tmp/mysql_connector.tar.gz ;fi
 ---> Using cache
 ---> a411998bee6f
Step 6/8 : ARG SOLR_DATAIMPORTHANDLER_MSSQL=false
 ---> Using cache
 ---> eac5f05e4041
Step 7/8 : ENV SOLR_DATAIMPORTHANDLER_MSSQL ${SOLR_DATAIMPORTHANDLER_MSSQL}
 ---> Using cache
 ---> c210a2cceb06
Step 8/8 : RUN if [ ${SOLR_DATAIMPORTHANDLER_MSSQL} = true ]; then     curl -L -o /tmp/mssql-jdbc-7.0.0.jre8.jar "https://github.com/Microsoft/mssql-jdbc/releases/download/v7.0.0/mssql-jdbc-7.0.0.jre8.jar"        && mkdir -p /opt/solr/contrib/dataimporthandler/lib        && mv /tmp/mssql-jdbc-7.0.0.jre8.jar "/opt/solr/contrib/dataimporthandler/lib/mssql-jdbc-7.0.0.jre8.jar" ;fi
 ---> Using cache
 ---> d966f5fd03b3
Successfully built d966f5fd03b3
Successfully tagged laradock_solr:latest
Building thumbor
Step 1/3 : FROM apsl/thumbor
 ---> 0cebbf43aedf
Step 2/3 : CMD ["thumbor"]
 ---> Using cache
 ---> bfe8b40b2134
Step 3/3 : EXPOSE 8000
 ---> Using cache
 ---> 1b8a556fe50b
Successfully built 1b8a556fe50b
Successfully tagged laradock_thumbor:latest
Building portainer
Step 1/2 : FROM portainer/portainer
 ---> ff4ee4caaa23
Step 2/2 : LABEL maintainer="luciano@lucianojr.com.br"
 ---> Using cache
 ---> acc74e31ff7c
Successfully built acc74e31ff7c
Successfully tagged laradock_portainer:latest
Building gitlab
Step 1/2 : FROM gitlab/gitlab-ce:latest
 ---> 5737e50aaa01
Step 2/2 : LABEL maintainer="ahkui <ahkui@outlook.com>"
 ---> Using cache
 ---> 0ee864307f95
Successfully built 0ee864307f95
Successfully tagged laradock_gitlab:latest
Building jupyterhub-user
Step 1/18 : FROM tensorflow/tensorflow:latest-gpu
 ---> 6cedbf5b8ce4
Step 2/18 : MAINTAINER ahkui <ahkui@outlook.com>
 ---> Using cache
 ---> 34cca0ca866a
Step 3/18 : RUN apt-get update && apt-get install -y --no-install-recommends         python         python-dev         &&     apt-get autoremove -y &&     apt-get autoclean &&     apt-get clean &&     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 ---> Using cache
 ---> 9198d258ba01
Step 4/18 : RUN apt-get update && apt-get install -y --no-install-recommends         wget         git         &&     apt-get autoremove -y &&     apt-get autoclean &&     apt-get clean &&     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 ---> Using cache
 ---> 9eb67248f012
Step 5/18 : RUN curl -O https://bootstrap.pypa.io/get-pip.py &&     python3 get-pip.py &&     rm get-pip.py
 ---> Running in f0adabc407aa
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 1734k  100 1734k    0     0   921k      0  0:00:01  0:00:01 --:--:--  926k
Traceback (most recent call last):
  File "get-pip.py", line 22318, in <module>
    main()
  File "get-pip.py", line 198, in main
    bootstrap(tmpdir=tmpdir)
  File "get-pip.py", line 82, in bootstrap
    import pip._internal.main
  File "/tmp/tmp1v2fjyf2/pip.zip/pip/_internal/main.py", line 13, in <module>
  File "/tmp/tmp1v2fjyf2/pip.zip/pip/_internal/cli/autocompletion.py", line 11, in <module>
  File "/tmp/tmp1v2fjyf2/pip.zip/pip/_internal/cli/main_parser.py", line 7, in <module>
  File "/tmp/tmp1v2fjyf2/pip.zip/pip/_internal/cli/cmdoptions.py", line 19, in <module>
ModuleNotFoundError: No module named 'distutils.util'
ERROR: Service 'jupyterhub-user' failed to build: The command '/bin/bash -c curl -O https://bootstrap.pypa.io/get-pip.py &&     python3 get-pip.py &&     rm get-pip.py' returned a non-zero code: 1
`

**Comment by larstbone** [comment]
Is [this an explanation](https://github.com/pypa/get-pip/issues/43) of the issue?

> For clarity for folks dealing with this, this occurs because someone over on the Debian side decided to actively exclude distutils from the default python.

If so, what is the work-around for Laradock?

**Comment by fuzzyjared** [comment]
The workaround for me was to add python3-distutils into the jupyterhub dockerfile.

This allowed it to progress in the build, however the build then failed when trying to install tensorflow GPU.

ERROR: tensorflow_gpu-1.8.0-cp35-cp35m-linux_x86_64.whl is not a supported wheel on this platform.

In my case I tried alternative wheels but wasn't able to get it to work, likely due to a hardware incompatibility isssue.

As I wasn't going to be using tensorflow I commented out that part of the Dockerfile and it was able to proceed. I suspect that the issue with tensorflow is due to Mac support for gpu. But not sure exactly how all that is routed in docker environments.
